apps/vaporgui/PImportDataButton.cpp

```diff
@@ -34,7 +34,7 @@ void PImportDataButton::_importDataset() {
     if (fileNames.empty()) return;
 
     std::string format = GetDatasets()[getParams()->GetValueLong(GUIStateParams::ImportDataTypeTag, 0)].first;
-    _mf->ImportDataset(fileNames, format);
+    _di->ImportDataset(fileNames, format);
```

Comment:
There are a few objects that are prevalent enough to have a universally understood shorthand name, such as `ce`, however, for general variables, names such as `cc` and `di` are ambiguous.